js/src/forum/components/PostStream.js

```diff
@@ -476,7 +476,7 @@ class PostStream extends Component {
       const top = $item.offset().top;
       const height = $item.outerHeight(true);
 
-      if (top + height > scrollTop) {
+      if (top + height > scrollTop + 2) {
```

Comment:
Could you explain/document where the number 2 is coming from?

Reply:
Sure. jQuery offset() and outerheight() can return value in fractions.
Instead of 5 , it may return 5.9. Since this happens twice, increasing value of scrolltop by 2, nullifies this.

Reply:
That makes sense, thanks! I think it would be good to have that explanation as a comment above this block of code so that it's instantly understandable to developers in the future.

Reply:
Most welcome! I have added a commit with a comment in code. Please let me know if its okay.

Reply:
Awesome!